types/react/index.d.ts

```diff
@@ -1688,7 +1688,7 @@ declare namespace React {
          * Indicates what notifications the user agent will trigger when the accessibility tree within a live region is modified.
          * @see aria-atomic.
          */
-        'aria-relevant'?: 'additions' | 'additions text' | 'all' | 'removals' | 'text';
+        'aria-relevant'?: 'additions' | 'additions removals' | 'additions text' | 'all' | 'removals' | 'removals text' | 'text';
```

Comment:
what about other orderings that are valid e.g. `removals additions`, `text removals`, `text additions`?  While we're hear shouldn't we include those as well?  How about the three member variations `additions removals text` (and all, what is it... 8? order permutations of those)?

Reply:
`all` represents all permutations of `additions removals text` so I don't think it's actually necessary to add those. Good point on the others though, I have updated them.

Reply:
cool